src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDirectiveDescriptor.cs

```diff
@@ -11,23 +11,24 @@ namespace Microsoft.AspNet.Razor.TagHelpers
     public class TagHelperDirectiveDescriptor
     {
         // Internal for testing purposes.
-        internal TagHelperDirectiveDescriptor(string lookupText,
+        internal TagHelperDirectiveDescriptor(string directiveText,
```

Comment:
admit I may be incorrect about `TagHelperDirective`.  but at this point you're dealing with separate concepts that are handled on distinct code paths in the consuming code.  no longer dealing with format at all.